df3d/core.py

```diff
@@ -412,7 +424,7 @@ def setup_camera_ordering(self, camera_ordering) -> np.ndarray:
 
     def expand_videos(self):
         """expands video camera_x.mp4 into set of images camera_x_img_y.jpg"""
-        for vid in glob.glob(os.path.join(self.input_folder, "camera_*.mp4")):
+        for vid in glob.glob(os.path.join(self.input_folder, "camera_?.mp4")):
```

Comment:
I guess this limits us to only using up to 10 cameras, but from what I could tell that we use 7 cameras is quite baked in so I think this makes sense